module/awesomebooks.js

```diff
@@ -0,0 +1,59 @@
+// check if the localStorage has a item
+let collections = JSON.parse(localStorage.getItem('books-collection')) || [];
+
+// create collection as page load
+const createCollection = (book, bookContainer) => {
+  const content = `
+    <article class="book">
+      <p>"${book.name.charAt(0).toUpperCase() + book.name.slice(1)}" by ${book.author.charAt(0).toUpperCase() + book.author.slice(1)}</p>
+      <button id="${book.id}">Remove</button>
+    </article>
+  `;
+  // append to the DOM
+  bookContainer.insertAdjacentHTML('beforeend', content);
+};
+
+// add event listener to button
+const removeBook = (id) => {
+  const btn = document.getElementById(`${id}`);
+
+  // onclick remove item from localStorage and collections
+  btn.addEventListener('click', (event) => {
+    event.target.parentElement.remove();
+    collections = collections.filter((book) => book.id !== id);
+    // add filtered collections to localStorage
+    localStorage.setItem('books-collection', JSON.stringify(collections));
+  });
+};
+
+export const displayBooks = (booksContainer) => {
+  collections.forEach((book) => {
+    createCollection(book, booksContainer);
+    removeBook(book.id);
+  });
+};
+
+export const addFormEventListener = (form, booksContainer) => {
```

Comment:
- [ ] [optional] It would have been nice if you added this add book function as an individual module, this will enable easier reading and scalability of your code.